sdk/eventhub/azure-eventhub/azure/eventhub/pyamqp/__init__.py

```diff
@@ -10,4 +10,4 @@
 from ._connection import Connection
 from ._transport import SSLTransport
 
-from .client import ReceiveClient, SendClient
+from .client import AMQPClient, ReceiveClient, SendClient
```

Comment:
Why is `pyamqp` public? This module should be private?

Reply:
agreed, I will make it private